terraform/groups/lambda/variables.tf

```diff
@@ -0,0 +1,55 @@
+variable "aws_region" {
```

Comment:
We need types on these

Reply:
Done. Added types to all variables in all modules.